firstvoices/backend/serializers/site_serializers.py

```diff
@@ -17,25 +17,16 @@
 from backend.serializers.widget_serializers import SiteWidgetListSerializer
 
 
-class FeatureFlagSerializer(serializers.Serializer):
-    id = serializers.CharField()
-    key = serializers.CharField()
-    is_enabled = serializers.BooleanField()
-
-
 class SiteSummarySerializer(LinkedSiteSerializer):
     """
     Serializes public, non-access-controlled information about a site object. This includes the type of info
     required for an "Explore Languages" type interface.
     """
 
     logo = ImageSerializer()
-    features = FeatureFlagSerializer(
-        read_only=True, source="sitefeature_set", many=True
-    )
 
     class Meta(LinkedSiteSerializer.Meta):
-        fields = LinkedSiteSerializer.Meta.fields + ("logo", "features")
+        fields = LinkedSiteSerializer.Meta.fields + ("logo",)
```

Comment:
Let's not remove this please-- it is a specially formatted list of only the enabled features for a site, currently used by the frontend for things like immersion mode

Reply:
Maybe we could rename it enabledFeatures or something, but we need to let Guy know.

Reply:
I have re-added the features field to the site serializer as enabledFeatures and I let Guy know about the change!